packages/optimizer/lib/transformers/AutoExtensionImporter.js

```diff
@@ -60,6 +60,7 @@ class AutoExtensionImporter {
     this.format = config.format || DEFAULT_FORMAT;
     this.log_ = config.log.tag('AutoExtensionImporter');
     this.experimentBindAttributeEnabled = config.experimentBindAttribute === true;
+    this.extensionVersions = config.extensionVersions || {};
```

Comment:
Theoretically, we could validate the provided parameter by: 

1. matching the extensions against the ones on the validator rules
2. checking if the version number is supported by the extension

Reply:
I would argue this shouldn't be validated, at least not in terms of ensuring the version number is supported. If I was using an invalid version number then either I'm doing it for a specific reason, or its clearly my fault and I should be able to debug from seeing the script 404 in my browser. In the case where I am doing it for a specific reason (e.g. playing with self hosting amp scripts) then I wouldn't want to deal with validation errors, I would expect the code to be "dumb" and just do what I tell it.

Reply:
sgtm